packages/ui/components/NavBar.vue

```diff
@@ -125,17 +125,7 @@ export default {
   align-items: center;
   background-color: var(--bg);
   height: 100px;
-  button,
```

Comment:
Is removing this related with this error?
![image](https://user-images.githubusercontent.com/24905681/189374208-2b181cdf-69d2-4f92-9a72-7f6e39dbff39.png)